src/types.ts

```diff
@@ -111,6 +112,7 @@ export type TInternalConfig = {
   extraLogoutParameters?: TPrimitiveRecord
   tokenExpiresIn?: number
   refreshTokenExpiresIn?: number
+  refreshTokenExpiryStrategy?: 'renewable' | 'absolute'
```

Comment:
```suggestion
  refreshTokenExpiryStrategy: 'renewable' | 'absolute'
```
No need to make this optional internally